azure-pipelines/cancel_aml_jobs.yml

```diff
@@ -0,0 +1,27 @@
+steps:
+  - checkout: self
+
+  - template: prepare_conda.yml
+
+  # https://docs.microsoft.com/en-us/azure/devops/pipelines/release/caching?view=azure-devops#pythonanaconda
+  - task: Cache@2
+    displayName: Use cached Conda environment AzureML_SDK
+    inputs:
+      # Beware of changing the cache key or path independently, safest to change in sync
+      key: 'usr_share_miniconda_azureml_conda | "$(Agent.OS)" | azure-pipelines/azureml-conda.yml'
+      path: /usr/share/miniconda/envs
+      cacheHitVar: CONDA_CACHE_RESTORED
+
+  - script: conda env create --file azure-pipelines/azureml-conda.yml
```

Comment:
Very minor suggestion to rename `azureml-conda.yml` to clarify it refers to a conda environment file rather than another build file (e.g. `azureml-conda-environment.yml` or so).